src/test/resources/liberty-test/install_liberty_webProfile7.gradle

```diff
@@ -19,7 +19,7 @@ repositories {
 }
 
 dependencies {
-    libertyRuntime group: 'com.ibm.websphere.appserver.runtime', name: 'wlp-webProfile7', version: runtimeVersion
```

Comment:
Reason for changing this?

Reply:
To test that the property specified in gradle.properties overrides the libertyRuntime name specified here.

Reply:
Oh gotcha, missed that.